# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1649443973 -7200
# Parent  515cd0ae5d9e21c198bbae39039e1697f08e72b0
Misc: Disable multi-lingual UI since there are no language packs for BB.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -862,17 +862,17 @@ pref("intl.regional_prefs.use_os_locales
 // Multi-lingual preferences:
 //  *.enabled - Are langpacks available for the build of Firefox?
 //  *.downloadEnabled - Langpacks are allowed to be downloaded from ATN. ATN only serves
 //      langpacks for release and beta. There is no release-only define, so we also enable
 //      it for beta.
 //  *.liveReload - Switching a langpack will change the language without a restart.
 //  *.liveReloadBidirectional - Allows switching when moving between LTR and RTL
 //      languages without a full restart.
-pref("intl.multilingual.enabled", true);
+pref("intl.multilingual.enabled", false); // No language packs for Betterbird so far.
 #if defined(RELEASE_OR_BETA)
 pref("intl.multilingual.downloadEnabled", true);
 pref("intl.multilingual.liveReload", false);
 pref("intl.multilingual.liveReloadBidirectional", false);
 #else
 pref("intl.multilingual.downloadEnabled", false);
 pref("intl.multilingual.liveReload", false);
 pref("intl.multilingual.liveReloadBidirectional", false);
